spec/spec.md

```diff
@@ -133,12 +133,12 @@ The following DID Document Service Endpoint entries ****MUST**** be present in t
 
 ```json
 {
-  "id": "did:example:123",
+  "id": "did:example:alice",
   "service": [{
     "id":"#dwn",
     "type": "DecentralizedWebNode",
     "serviceEndpoint": {
-      "nodes": ["https://dwn.example.com", "https://example.org/dwn"]
+      "nodes": ["did:example:host", "https://dwn.example.com"]
```

Comment:
@csuwildcat looking at https://github.com/decentralized-identity/decentralized-web-node/blob/dc69c89ffb12eb304f796f368f65b78e172d8826/spec/spec.md?plain=1#L202, you mention it resolves to "https://dwn.example.com". Per this example, given the preference toward resolving toward the first listed node in the service endpoint section, wouldn't it resolve to did:example:host first, which may or may not resolve eventually to `https://dwn.example.com`? Wouldn't it make more sense to list `https://dwn.example.com` first, so the reference below is more consistent?